docs/operator-manual/airgap/02-install.md

```diff
@@ -0,0 +1,155 @@
+---
+sidebar_label: "Installation"
+title: ""
+---
+
+# Air gap installation
+
+This guide will show you how to install Kubewarden in air gap environments. In an air gap installation of Kubewarden, 
+you will need a private OCI registry that is located somewhere accessible by your kubernetes cluster. Kubewarden Policies 
+are WebAssembly modules, therefore they can be stored inside of an OCI compliant registry as OCI artifacts.
+You need to add Kubewarden's images and policies to this OCI registry. Let's see how to do that.
+
+## Save container images in your workstation
+
+1. Download `kubewarden-images.txt` from the Kubewarden [release page](https://github.com/kubewarden/helm-charts/releases/). Alternatively, you can find `imagelist.txt` and `policylist.txt` files shipped inside the helm charts, that contain the used container images and policy wasm modules respectively.
+:::note
+Optionally, you can verify the signatures of the [helm charts](../../security/verifying-kubewarden#helm-charts) and [container images](../../security/verifying-kubewarden#container-images)
+:::
+2. Add `cert-manager` if it is not available in your private registry. 
+```
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm pull jetstack/cert-manager --version v1.7.1
```

Comment:
You are defining a version here. But in the command to install the cert-manager you does not define the version. Should we remove the version here as well?

Reply:
done https://github.com/kubewarden/docs/pull/150/commits/5df0705109afd4b2280d2462080ad48b92b9e9ad